src/extension.ts

```diff
@@ -191,6 +196,11 @@ export function activate(context) {
     subscriptions.forEach((element) => {
         context.subscriptions.push(element);
     }, this);
+
+    return {
+        apiVersion: '0.1',
+        clusterProviderRegistry: clusterProviderRegistry
+    };
```

Comment:
can you talk about what this does? never seen this pattern of returning an object from the activate function before

Reply:
The return value of `activate` is an object that other extensions can interact with if they call `getExtension<T>` and `extension<T>.activate`.  So it acts as a way to surface an API to your extension.  https://code.visualstudio.com/docs/extensionAPI/vscode-api#_extensions